json-rpc/README.md

```diff
@@ -116,16 +116,24 @@ operators such as `Init`, `Next`, and the invariants.
     "sessionId": "unique-session-identifier",
     "snapshotId": snapshot-identifier-after-loading-the-spec,
     "specParameters": {
-      "nInitTransitions": number-of-Init-transitions,
-      "nNextTransitions": number-of-Next-transitions,
-      "nStateInvariants": number-of-state-invariants,
-      "nActionInvariants": number-of-action-invariants,
-      "nTraceInvariants": number-of-trace-invariants
+      "initTransitions": init-metadata,
+      "nextTransitions": next-metadata,
+      "stateInvariants": state-invariants-metadata,
+      "actionInvariants": action-invariants-metadata,
```

Comment:
it becomes apparent in the example, but it would be good to clarify that the metadata entries are JSON arrays